cranelift/reader/src/lexer.rs

```diff
@@ -491,6 +493,10 @@ impl<'a> Lexer<'a> {
                 Some('a'..='z') | Some('A'..='Z') => {
                     if self.looking_at("NaN") || self.looking_at("Inf") {
                         Some(self.scan_number())
+                    } else if self.looking_at("then") {
+                        Some(self.scan_chars(4, Token::Then))
+                    } else if self.looking_at("else") {
+                        Some(self.scan_chars(4, Token::Else))
```

Comment:
I haven't looked at how this lexer works. Does this correctly handle missing whitespace? Like, if someone wrote `brif v0 thenblock1 elseblock2`, would it reject that or would it strip off the "then"/"else" prefix and then see that it has a valid block name?

Reply:
You're right, it would parse `thenblock0`. I've opted to remove those keywords for now, separating the control argument and blocks with commas instead:
```
brif v1, block1, block2
```